app/models/user.rb

```diff
@@ -1,6 +1,41 @@
 class User < ActiveRecord::Base
+  before_create :grow_beard
+  before_save :hash_password
 
   def fullname
     first_name + " " + last_name
   end
+
+  def trim
+    self.status = "Trimmed"
+    self.save
+  end
+
+  def massage_and_trim
+    self.status = "Massaged and Trimmed"
+    self.save
+  end
+
+  def shave
+    self.status = "Shaved"
+    self.save
+  end
+
+  def grow_beard
+    self.status = "Unshaved beard"
```

Comment:
Your usage of string constants in the ```status``` attribute is inconsistent and this will lead to bugs.  Here you have used "Unshaved beard" whereas above you have used "Massaged and Trimmed".  It is more standard practice to use either underscore ```massaged_and_trimmed`` or snake case ```massagedAndTrimmed`` for rails applications it's better to use underscoring because ruby style is underscores.